olwebsite-app/src/pages/PostsView.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import SearchBox from "../components/SearchBox/SearchBox";
+
+const handleSearch = searchQuery => {
+  // TODO - the searching function should beware of empty strings
+  alert(searchQuery);
+};
+
+const PostsView = () => {
+  return (
+    <div className="posts-view-container">
+      <h1>Posts View</h1> <br />
```

Comment:
do we need this?

Reply:
I got the Posts page from the figma designs.

Reply:
I meant do we need the <br /> tag, shouldve specified